data/libs/prelude.icicle

```diff
@@ -6,11 +6,48 @@ mean v = sum (double v) / count.
 max v = let fold1 s = v : case v > s | True -> v | False -> s end ~> s.
 min v = let fold1 s = v : case v < s | True -> v | False -> s end ~> s.
 
+minby a b = let fold1 s = (a, b)
+   :  case s | (a1, b1) -> (case a < a1 | True -> (a,b) | False -> (a1,b1) end) end
+   ~> case s | (a1, b1) -> b1 end.
+
+maxby a b = let fold1 s = (a, b)
+   :  case s | (a1, b1) -> (case a > a1 | True -> (a,b) | False -> (a1,b1) end) end
+   ~> case s | (a1, b1) -> b1 end.
+
+# Numerically unstable standard deviation. Best to not use.
 sd_old v = let v_ = double v ~> let sv = (sum v_) ~> (((sum (v_*v_) * count) - (sv * sv)) / (count * (count - 1)))^(0.5).
```

Comment:
I think we can just remove this and "update" any test that might use this `sd_old`?